base/operators.jl

```diff
@@ -3,10 +3,27 @@
 ## types ##
 
 """
-    <:(T1, T2)
+```julia
+<:(L, R)
+```
+
+The subtyping predicate: test whether the type `L` may be a subtype of `R`.
+
+For any type `A` and any type `B`, `A` subtypes `B` if and only if any value
+belonging to `A` is also of type `B`. Subtyping is reflexive and transitive,
+making it a *preorder*.
```

Comment:
`<:` is also antisymmetric, by definition of `==`, which is defined in terms of `<:`. So (if we actually had transitivity, see #53019 and #53021) we could actually use the stronger label of *partial order*